verify/trust/trust.go

```diff
@@ -89,12 +127,17 @@ func (n *RetryHTTPSGetter) Get(url string) (map[string][]string, []byte, error)
 		select {
 		case <-ctx.Done():
 			cancel()
-			return nil, nil, fmt.Errorf("timeout") // context cancelled
+			return nil, nil, ctx.Err()
 		case <-time.After(delay): // wait to retry
 		}
 	}
 }
 
+var (
+	_ = HTTPSGetter(&RetryHTTPSGetter{})
```

Comment:
I don’t understand

Reply:
These statements ensure that `RetryHTTPSGetter` implements both `HTTPSGetter` and `ContextHTTPSGetter`. It's not strictly necessary and I', open to removing them, but I think it helps prevent accidental divergence, especially in the case of `ContextHTTPSGetter` which is only tested for indirectly.

Reply:
Better to put this kind of check in a test.